docs/release-notes/1732-feat-more-flexible-local-gradient-agg.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**IMPROVEMENTS**
+
+-  PyTorchTrial: support more flexible local gradient aggregation 
```

Comment:
blocking: I think we should be more specific in the use cases like supporting more than 1 times local gradient aggregation for each optimizer